### PR TITLE
Adding VTOL Octoplane airframe type

### PR DESCRIPTION
**Pull request by irsdkv**
At this time some VTOLs could have 8 motors for MC mode.
I think that the ability to develop their support will be more convenient with the appropriate class.

Corresponding PR in Firmware: https://github.com/PX4/Firmware/pull/12303
Corresponding PR in Devguide: https://github.com/PX4/Devguide/pull/818
Corresponding PR in QGroundcontrol: https://github.com/mavlink/qgroundcontrol/pull/7537

### CODE REVIEW

File: en/airframes/airframe_reference.md [hunk 1]
```diff
@@ -1052,3 +1052,21 @@ This page lists all supported airframes and types including
 </tr>
 </tbody></table>
 
+
```
Comment (hamishwillee) [inline en/airframes/airframe_reference.md]: Please delete this file/change - this file is autogenerated from the work you did in Firmware repo. It is already there: https://dev.px4.io/master/en/airframes/airframe_reference.html#vtol-octoplane

But you do need to add the image :-)

Comment (irsdkv) [reply en/airframes/airframe_reference.md]: Oh, sure :)

**Comment by hamishwillee** [comment]
THanks! Merged

**Comment by hamishwillee** [comment]
And looks good https://dev.px4.io/master/en/airframes/airframe_reference.html#vtol-octoplane

**Pull request merged**